server/Serials/Lib/Crud.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE OverloadedStrings #-}
 
-module Serials.Model.Crud where
+module Serials.Lib.Crud where
```

Comment:
No I didn't think so, but I don't care what it is called other then Crud does not feel like a model at all. So if you have other suggestions I am open. Unless you want it in Model, but that feels wrong to me.